copilot.sh

```diff
@@ -5,8 +5,13 @@ set -o errexit
 set -o nounset
 
 # This script requires https://github.com/charmbracelet/gum
-command -v /usr/bin/gum >/dev/null 2>&1 \
-  || { echo >&2 "This script requires https://github.com/charmbracelet/gum, but it's not installed.  Aborting."; exit 1; }
+# gum can is typically installed in /usr/bin/gum or /usr/local/bin/gum 
+check_installed=$(command -v gum)
+
+if [ -z "$check_installed" ]; then
+
+  echo "This script requires https://github.com/charmbracelet/gum, but it's not installed.  Aborting."; exit 1; 
+fi  
 
 # This script requires https://github.com/martindstone/pagerduty-cli
 command -v /usr/local/bin/pd >/dev/null 2>&1 \
```

Comment:
This line should be handled the same way (`command -v pd`).

Reply:
I reverted, thanks for taking a look!